src/vs/workbench/parts/output/electron-browser/outputServices.ts

```diff
@@ -612,9 +601,7 @@ export class OutputService extends Disposable implements IOutputService, ITextMo
 					if (!preserveFocus) {
 						this._outputPanel.focus();
 					}
-				})
```

Comment:
Calling this function doesn't make sense anymore. If the lock was set before we switched the channel it should stay set. We shouldn't trigger unsetting. (this was different when the "smart ' behavior was independent of the lock.